src/functions/template-tags/google-map.php

```diff
@@ -24,8 +24,8 @@
 	 * @return string A fully qualified link to http://maps.google.com/ for this event
 	 */
 	function tribe_get_map_link( $postId = null ) {
-		$tribe_ecp = Tribe__Events__Main::instance();
-		$output    = esc_url( $tribe_ecp->googleMapLink( $postId ) );
+		$tec = Tribe__Events__Main::instance();
```

Comment:
Should either fix alignment of `=` on this line or drop alignment on the following line.